bot/exts/events/advent_of_code/views/dayandstarview.py

```diff
@@ -42,7 +42,10 @@ def generate_output(self) -> str:
 
     async def interaction_check(self, interaction: discord.Interaction) -> bool:
         """Global check to ensure that the interacting user is the user who invoked the command originally."""
-        return interaction.user == self.original_author
+        if interaction.user != self.original_author:
+            await interaction.response.send_message(":x: This is not your view to interact with!", ephemeral=True)
```

Comment:
My only comment is that while this message makes sense for someone familiar with interactions and discord.py, it may not be clear to those who don't know what a "view" means.

We could maybe reword this to something like: "You can't interact with someone else's response. Please run the command yourself."

Reply:
That's a valid point.
I've pushed a commit that makes the error message a bit more understandable for everyone!